pages/linux/udevadm.md

```diff
@@ -0,0 +1,20 @@
+# udevadm
+
+> Linux `udev` management tool.
+> More information: <https://linux.die.net/man/8/udevadm>.
+
+- Monitor device events:
+
+`sudo udevadm monitor [--kernel] [--udev]`
```

Comment:
We don't use `[]` for examples, I think you should split this example to explain the command without params, another for `--kernel` and another for `--udev`.